bundler.d/development.rb

```diff
@@ -4,7 +4,7 @@
   gem 'rubocop', '0.26.1'
 
   # for generating i18n files
-  gem 'gettext', '~> 3.1', :require => false
+  gem 'gettext', '>= 3.1.3', '< 4.0.0', :require => false
```

Comment:
development.rb is the correct location, assets.rb is the duplicate.  It has a comment above explaining, but it's a missing dependency in gettext_i18n_rails_js on gettext, so if somebody didn't have the development group, they'd need the gettext dep too.